client/main.lua

```diff
@@ -256,96 +290,108 @@ end)
 
 RegisterNetEvent("qb-diving:client:setoxygenlevel", function()
     if oxgenlevell == 0 then
-       oxgenlevell = 100 -- oxygenlevel
-       QBCore.Functions.Notify(Lang:t("success.tube_filled"), 'success')
-       TriggerServerEvent('qb-diving:server:removeItemAfterFill')
+        oxgenlevell = 100 -- oxygenlevel
+
+        QBCore.Functions.Notify(Lang:t("success.tube_filled"), 'success')
+
+        TriggerServerEvent('qb-diving:server:removeItemAfterFill')
     else
         QBCore.Functions.Notify(Lang:t("error.oxygenlevel", {oxygenlevel = oxgenlevell}), 'error')
     end
 end)
```

Comment:
If you are gonna expand this out for readability, wouldn't turning this into a guard clause be better solution.
```lua
if oxgenlevell ~= 0 then
     QBCore.Functions.Notify(Lang:t("error.oxygenlevel", {oxygenlevel = oxgenlevell}), 'error')
     return
end

oxgenlevell = 100 -- oxygenlevel
QBCore.Functions.Notify(Lang:t("success.tube_filled"), 'success')
TriggerServerEvent('qb-diving:server:removeItemAfterFill')
```
I think this is infinitely more readable